.appveyor.yml

```diff
@@ -98,7 +98,7 @@ for:
       - copy C:\Python38-x64\python.exe C:\Python38-x64\python3.exe
       - python3 -m pip install "git+https://github.com/rizinorg/rz-pipe#egg=rzpipe&subdirectory=python"
       - cd test
-      - git clone -q --depth 1 https://github.com/rizinorg/rizin-testbins bins
+      - git clone -q --depth 1 -c core.symlinks=true https://github.com/rizinorg/rizin-testbins bins
```

Comment:
actually... would this work on windows?

Reply:
Looks like it is: https://ci.appveyor.com/project/rizinorg/rizin/builds/42313457/job/l4l1nchu005kbc2u?fullLog=true#L3735

Reply:
apparently it requires the user to be an Administrator but if the user is not, currently only 1 test is affected and if that turns out to be a problem, it is not hard to modify the test so that it keeps fulfilling its original objective whether or not the user is an Administrator